api/EventTarget.json

```diff
@@ -0,0 +1,507 @@
+                {
+                  "version_added": "50",
+                  "notes": "<code>once</code> option supported."
+                },
+                {
+                  "version_added": "49"
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "50",
+                  "notes": "<code>once</code> option supported."
+                },
+                {
+                  "version_added": "49"
+                }
+              ],
+              "ie": {
+                "version_added": false
+              },
+              "opera": [
+                {
+                  "version_added": "42",
+                  "notes": "<code>once</code> option supported."
+                },
+                {
+                  "version_added": true
+                }
+              ],
+              "opera_android": [
+                {
+                  "version_added": "42",
+                  "notes": "<code>once</code> option supported."
+                },
+                {
+                  "version_added": true
+                }
+              ],
+              "safari": {
+                "version_added": "10"
+              },
+              "safari_ios": {
+                "version_added": "10"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        }
+      },
+      "dispatchEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/EventTarget/dispatchEvent",
+          "support": {
+            "webview_android": {
+              "version_added": "4"
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "4"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "2"
+            },
+            "firefox_android": {
+              "version_added": "2"
+            },
+            "ie": {
+              "version_added": "9",
+              "notes": "Older versions of IE supported an equivalent, proprietary, <code>EventTarget.fireEvent()</code> method."
+            },
+            "opera": {
+              "version_added": "51"
+            },
+            "opera_android": {
+              "version_added": "51"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "removeEventListener": {
```

Comment:
I would mirror the table of `addEventListener` here. See my comments above.

Reply:
According to the removeEventListener page, that complexity isn't needed; the support info is simpler. So I've not made quite as many changes here.